selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -607,4 +607,31 @@ public String getModel() {
   public String getAPITargetType() {
     return apiTargetType;
   }
+
+  public void unlockScreen() throws AndroidDeviceException {
+    // Get phone's android version and whether the screen is off.
```

Comment:
I agree this method should be defined in AbstractDevice, but the contents you have here should go into DefaultHardwareDevice

and then the method unlockEmulatorScreen in DefaultAndroidEmulator should get renamed.